simulariumio/nerdss/nerdss_converter.py

```diff
@@ -0,0 +1,168 @@
+from MDAnalysis import Universe
+import os
+
+from ..trajectory_converter import TrajectoryConverter
+from ..data_objects import (
+    AgentData,
+    TrajectoryData,
+    DimensionData,
+    UnitData,
+    DisplayData,
+)
+from ..constants import DISPLAY_TYPE, VALUES_PER_3D_POINT, VIZ_TYPE
+from .nerdss_data import NerdssData
+
+
+class NerdssConverter(TrajectoryConverter):
+    def __init__(
+        self,
+        input_data: NerdssData,
+    ):
+        """
+        Parameters
+        ----------
+        input_data : NerdssData
+            An object containing info for reading
+            NERDSS simulation trajectory outputs and plot data
+        """
+        self._data = self._read(input_data)
+
+    def _read_pdb_files(self, input_data: NerdssData) -> AgentData:
+        file_list = os.listdir(input_data.path_to_pdb_files)
+        file_list.sort()
+        n_timesteps = len(file_list)
+        dimensions = DimensionData(
+            total_steps=n_timesteps, max_agents=0, max_subpoints=VALUES_PER_3D_POINT * 2
+        )
+        time_steps = []
+        for file in file_list:
+            u = Universe(os.path.join(input_data.path_to_pdb_files, file))
+            n_agents = len(u.atoms.positions)
```

Comment:
one agent per atom position?  just curious, why not treat them as standalone pdb agents?  is this just to be able to draw the "fiber" bonds?

It looks to me like the pdbs are going to be actually different on every time step (is that true?).  If so, it's going to result in a whole lot of agents either way. Maybe then it's actually better to use spheres like this, rather than cache entire independent pdbs in the browser.